string name
pcl_msgs/PolygonMesh mesh